controllers/managedocs_controller.go

```diff
@@ -722,6 +731,39 @@ func (r *ManagedOCSReconciler) getDesiredConvergedStorageCluster() (*ocsv1.Stora
 	return sc, nil
 }
 
+func (r *ManagedOCSReconciler) getDesiredConsumerStorageCluster() (*ocsv1.StorageCluster, error) {
+
+	capacityAsString := r.addonParams[consumerRequestedCapacity]
+	onboardingTicket := r.addonParams[consumerOnboardingTicket]
+	storageProviderEndpoint := r.addonParams[consumerStorageProviderEndpoint]
+
+	requestedCapacity, err := resource.ParseQuantity(capacityAsString)
+	if err != nil {
+		return nil, fmt.Errorf("Invalid storage cluster capacity value: %v", capacityAsString)
```

Comment:
the error message should not start with capital letters. use `errors.Wrapf` or add actual error message to the return